NextTime.yaml

```diff
@@ -19,4 +19,5 @@ suggestions:
   - Markup
   - CMake
   - WSL guide
+  - YAML work
```

Comment:
What does YAML work mean? Like how YAML files are formatted/parsed? Or how we use them?